### PR TITLE
[caldav-personal] Update README after filter change

### PR DESCRIPTION
**Pull request by openhab-5iver**
Updates filter examples to include changes in #5935.

Signed-off-by: Scott Rushworth <openhab@5iver.com>

### CODE REVIEW

File: bundles/binding/org.openhab.binding.caldav-personal/README.md [hunk 1]
```diff
@@ -82,29 +82,34 @@ Depending on your rule implementation, it is possible to use event entries like
...
-* Used to detect presence through calendar events.
-* Used to show upcoming/active events in openhab.
+* Used to detect presence through calendar events
+* Used to show upcoming/active events in openhab
 
 ### openhab.cfg
-* `caldavPersonal:usedCalendars=<calendar-id>` (multiple calendars can be seperated by commas)
-* `caldavPersonal:homeIdentifiers=<values seperated by commans>` (if one of these identifiers can be found inside the place of the event, this event will not be used for presence)
+* `caldavPersonal:usedCalendars=<calendar-id>` (multiple calendars can be separated by commas)
+* `caldavPersonal:homeIdentifiers=<values separated by commans>` (if one of these identifiers can be found inside the place of the event, this event will not be used for presence)
```
Comment (9037568) [inline bundles/binding/org.openhab.binding.caldav-personal/README.md]: Can you fix this `commans` typo?

**Comment by 9037568** [comment]
Great stuff, thanks for this.

File: bundles/binding/org.openhab.binding.caldav-personal/README.md (hunk 1 above)
Comment (openhab-5iver) [reply bundles/binding/org.openhab.binding.caldav-personal/README.md]: I've been working from the web interface and I'd rather submit another PR after this one is merged. OK?

Comment (9037568) [reply bundles/binding/org.openhab.binding.caldav-personal/README.md]: Not sure it's worth that, but if you want, I'm ok with it.

**Comment by openhab-5iver** [comment]
Closing due to repo no longer being maintained.

**Pull request closed without merging**